@vates/disposable/README.md

```diff
@@ -32,7 +32,7 @@ import { deduped } from '@vates/disposable/deduped'
 // the connection with the passed host will be established once at the first call, then, it will be shared with the next calls
 const getConnection = deduped(async function (host)) {
   const connection = new Connection(host)
-  return new Disposabe(connection, () => connection.close())
+  return new Disposable(connection, () => connection.close())
```

Comment:
I perform corrections across codebases, this change (https://github.com/vatesfr/xen-orchestra/pull/8593/commits/91bb10eac07ae9aa1653e56ebdad66fa7f862807) means that the documentation wasn't in sync with the [code](https://github.com/search?q=repo%3Avatesfr%2Fxen-orchestra%20Disposable&type=code), which is unfortunate. It's one of many reasons that I make PRs like this and advocate for checking spelling across both code and documentation.